client.go

```diff
@@ -52,6 +52,14 @@ const (
 
 type Timestamp time.Time
 
+// return by Capture for nil clients
+var closedChan chan error
```

Comment:
Sure, that's fine with me. Arguably since the "normal" code path also allocates a channel, this won't really be any different.